From 0fbfe3c6c088ee77af3f8ca6059b1817f2ec9efe Mon Sep 17 00:00:00 2001
From: Mathew McBride <matt@traverse.com.au>
Date: Mon, 16 Dec 2024 14:11:23 +1100
Subject: [PATCH 2/4] platform: raspberry-pi: remove RPI reference for default
 SSID naming

Our changes for RDK-B on generic Arm boards don't necessitate a
new platform/ folder yet, so use a simple patch in the mean time

Change-Id: Ibc6d1d8e373f1e648cd4534d986c96eeae77ca08
Signed-off-by: Mathew McBride <matt@traverse.com.au>
---
 platform/raspberry-pi/platform_pi.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/platform/raspberry-pi/platform_pi.c b/platform/raspberry-pi/platform_pi.c
index 3c5c892..aee573e 100644
--- a/platform/raspberry-pi/platform_pi.c
+++ b/platform/raspberry-pi/platform_pi.c
@@ -204,7 +204,7 @@ int platform_get_ssid_default(char *ssid, int vap_index)
             return 0;
         }
     }
-    sprintf(ssid, "RPI_RDKB-AP%d", vap_index);
+    sprintf(ssid, "RDKB-AP%d", vap_index);
     return 0;
 }
 
@@ -800,4 +800,4 @@ UINT wifi_freq_to_op_class(UINT freq)
     }
 
     return op_class;
-}
\ No newline at end of file
+}
-- 
2.39.5

